src/pages/todos/components/TodoList/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Dispatch, SetStateAction, useCallback } from 'react';
+
+import { TodoType } from '@pages/todos';
+import {
+  StyledTodoItem,
+  StyledTodoListItems,
+} from '@pages/todos/components/TodoList/TodoList.styles';
+import CheckBoxComponent from '@components/CheckBox';
+import ButtonComponent from '@components/Button';
+
+type TodoListProps = {
+  todos: TodoType[];
+  setTodos: Dispatch<SetStateAction<TodoType[]>>;
```

Comment:
avoid passing `setState` functions since it makes your code error prone, find a better name and pass a callback instead.